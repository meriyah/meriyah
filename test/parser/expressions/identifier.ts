import { pass } from '../../test-utils';

pass('Identifier', [{ code: String.raw`a\u0065`, options: { loc: true, ranges: true } }]);
