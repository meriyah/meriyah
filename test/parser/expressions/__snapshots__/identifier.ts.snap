// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Identifier > a\\u0065 1`] = `
{
  "body": [
    {
      "end": 7,
      "expression": {
        "end": 7,
        "loc": {
          "end": {
            "column": 7,
            "line": 1,
          },
          "start": {
            "column": 0,
            "line": 1,
          },
        },
        "name": "ae",
        "range": [
          0,
          7,
        ],
        "start": 0,
        "type": "Identifier",
      },
      "loc": {
        "end": {
          "column": 7,
          "line": 1,
        },
        "start": {
          "column": 0,
          "line": 1,
        },
      },
      "range": [
        0,
        7,
      ],
      "start": 0,
      "type": "ExpressionStatement",
    },
  ],
  "end": 7,
  "loc": {
    "end": {
      "column": 7,
      "line": 1,
    },
    "start": {
      "column": 0,
      "line": 1,
    },
  },
  "range": [
    0,
    7,
  ],
  "sourceType": "script",
  "start": 0,
  "type": "Program",
}
`;
